scripts/run_all_simulation.sh

```diff
@@ -10,6 +10,7 @@ echo "Note that this simulation will not work if connected to external nodes."
 
 _run_bg_cmd python -m src.server.start_harvester
 _run_bg_cmd python -m src.server.start_timelord
+_run_bg_cmd python -m src.timelord_launcher
```

Comment:
Have you tested these scripts?

Reply:
I've tested only run_all and run_all_simulation, however others should work exactly the same.